test/comments.d

```diff
@@ -220,7 +220,9 @@ unittest
 unittest
 {
     setAPIExpectations(
-        "/github/repos/dlang/phobos/pulls/4921/commits",
+        "/github/repos/dlang/phobos/pulls/4921/commits", (ref Json j) {
+            j[0]["commit"]["message"] = "Fix Issue 8573";
```

Comment:
The commit message was "Issue 8573 ..." so no stable warning would have been shown for this test.